check_yum

```diff
@@ -355,15 +358,18 @@ class YumTester:
 		if number_security_updates == 0:
 			status = OK
 			message = "0 Security Updates Available"
+			perfdata = "security_updates=0"
 		else:
 			if self.no_warn_on_updates:
 				status = OK
 			else:
 				status = CRITICAL
 			if number_security_updates == 1:
 				message = "1 Security Update Available"
+				perfdata = "security_updates=1"
 			elif number_security_updates > 1:
 				message = "%s Security Updates Available" % number_security_updates
+				perfdata = "security_updates=%s" % number_security_updates
```

Comment:
same here, setting it once after the if-clause should work just fine (doesn't it?) and makes the code more bloated